src/main/java/com/axway/apim/swagger/APIImportConfigAdapter.java

```diff
@@ -109,6 +113,18 @@ public IAPI getDesiredAPI() throws AppException {
 			} else {
 				stagedConfig = baseConfig;
 			}
+			Path currentRelativePath = Paths.get("");
+			String s = currentRelativePath.toAbsolutePath().toString();
+			LOG.info("path={}",s);
+			if (StringUtils.isEmpty(this.pathToAPIDefinition)) {
+				if (StringUtils.isNotEmpty(stagedConfig.getApiDefinitionImport())) {
+					this.pathToAPIDefinition=baseConfig.getApiDefinitionImport();
+					LOG.info("Reading API Definition from contract file");

```

Comment:
Some time ago, I have changed the word: "API Contract" into "API Configuration" as colleagues & customers asked me why this file is called contract. 
Can you please change it to config at all places?

Reply:
ok